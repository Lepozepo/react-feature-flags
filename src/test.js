import { FeatureFlagsConsumer } from './'

describe('FeatureFlagsConsumer', () => {
  it('is truthy', () => {
    expect(FeatureFlagsConsumer).toBeTruthy()
  })
})
