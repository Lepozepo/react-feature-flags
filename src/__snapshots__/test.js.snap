// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flags when flag is falsy it should return the component or element given by renderOff props when authorizedFlags match flags but aren't active 1`] = `
<h1>
  renderOff rendered
</h1>
`;

exports[`Flags when flag is truthy when exactFlags is falsy it should return the component or element given by children props when authorizedFlags match flags 1`] = `
<h1>
  children rendered
</h1>
`;

exports[`Flags when flag is truthy when exactFlags is falsy it should return the component or element given by renderOn props when authorizedFlags match flags 1`] = `
<h1>
  renderOn rendered
</h1>
`;

exports[`Flags when flag is truthy when exactFlags is truthy it should return the component or element given by renderOff props when authorizedFlags doesn't match exactly flags 1`] = `
<h1>
  renderOff rendered
</h1>
`;

exports[`Flags when flag is truthy when exactFlags is truthy it should return the component or element given by renderOff props when authorizedFlags match exactly flags but aren't all active 1`] = `
<h1>
  renderOff rendered
</h1>
`;

exports[`Flags when flag is truthy when exactFlags is truthy it should return the component or element given by renderOn props when authorizedFlags match exactly flags 1`] = `
<h1>
  renderOn rendered
</h1>
`;

exports[`FlagsProvider it should return the given children 1`] = `
<h1>
  whatever node
</h1>
`;
